### ISSUE TITLE
Bittrex crashes upon start

### ISSUE THREAD

**Issue by vaisov** [DeviaVir/zenbot#435]
I get the following error whenever I try to start trading on bittrex. I tried to run `npm install` and `npm update`, but that didn't solve the issue.
```
APIKEY_INVALID
APIKEY_INVALID
APIKEY_INVALID
APIKEY_INVALID
APIKEY_INVALID
APIKEY_INVALID
APIKEY_INVALID
APIKEY_INVALID
/home/ubuntu/zenbot/node_modules/numbro/numbro.js:1111
        var parts = x.toString().split('.');
                     ^

TypeError: Cannot read property 'toString' of undefined
    at multiplier (/home/ubuntu/zenbot/node_modules/numbro/numbro.js:1111:22)
    at /home/ubuntu/zenbot/node_modules/numbro/numbro.js:1127:22
    at Array.reduce (native)
    at correctionFactor (/home/ubuntu/zenbot/node_modules/numbro/numbro.js:1125:21)
    at Object.add (/home/ubuntu/zenbot/node_modules/numbro/numbro.js:1197:47)
    at /home/ubuntu/zenbot/commands/trade.js:234:78
    at /home/ubuntu/zenbot/lib/engine.js:188:13
    at /home/ubuntu/zenbot/extensions/exchanges/bittrex/exchange.js:165:9
    at Request._callback (/home/ubuntu/zenbot/node_modules/node.bittrex.api/node.bittrex.api.js:114:25)
    at Request.self.callback (/home/ubuntu/zenbot/node_modules/request/request.js:188:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/home/ubuntu/zenbot/node_modules/request/request.js:1171:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/home/ubuntu/zenbot/node_modules/request/request.js:1091:12)
    at IncomingMessage.g (events.js:292:16)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

**Comment by DeviaVir** [comment]
Did you set the right bittrex API key?

**Comment by vaisov** [comment]
Yep, I tried that with two different keys. One I tried with "Read Info", "Trade Limit" and "Trade Market" permissions, but then I read in conf.js, that it has to be only "Read Info" and "Trade Limit". So I created a new key with only these, but I still get the same error. According to "APIKEY_INVALID" it's quite obvious, that it could be wrong key, but I tried that with two new keys with no luck. Any other idea how could I debug this? I can create another key, but I doubt it's the key issue.
